en/docs/services/service-concepts.md

```diff
@@ -0,0 +1,113 @@
+
+   | Event                       | Action                 | Application Trigger                      |
+   |-----------------------------|------------------------|------------------------------------------|
+   | issue_comment               | created                | Add a comment to an issue                |
+   |                             | edited                 | Edit a comment on an issue               |
+   |                             | deleted                | Delete a comment on an issue             |
+   | issues                      | assigned               | Assign an issue to a user                |
+   |                             | unassigned             | Unassign an issue from a user            |
+   |                             | labeled                | Add a label to an issue                  |
+   |                             | unlabeled              | Remove a label from an issue             |
+   |                             | opened                 | Open an issue                            |
+   |                             | edited                 | edit an issue                            |
+   |                             | milestoned             | Add a milestone to an issue              |
+   |                             | demilestoned           | Remove a milestone from an issue         |
+   |                             | closed                 | Close an issue                           |
+   |                             | reopened               | Reopen an issue                          |
+   | label                       | created                | Create a label                           |
+   |                             | edited                 | Edit a label                             |
+   |                             | deleted                | Delete a label                           |
+   | milestone                   | created                | Create a milestone                       |
+   |                             | closed                 | Close a milestone                        |
+   |                             | opened                 | Open a milestone                         |
+   |                             | edited                 | Edit a milestone                         |
+   |                             | deleted                | Delete a milestone                       |
+   | pull_request                | assigned               | Assign a pull request to a user          |
+   |                             | unassigned             | Unassign a pull request from a user      |
+   |                             | review_requested       | Request a user to review a pull request  |
+   |                             | review_request_removed |                                          |
```

Comment:
`Application trigger` column is empty

Reply:
Removed this content as it is no longer applicable to services.